xopt/bayesian/acquisition/exploration.py

```diff
@@ -51,8 +50,10 @@ def __init__(
             elif isinstance(self.model, GPyTorchModel):
                 self.sigma = torch.eye(self.model.train_inputs[0].shape[-1]) * 1e6
             else:
-                raise NotImplementedError('Get Ryan to make corrections for this type of model in proximal acq, '
-                                          'or specify your own sigma matrix')
+                raise NotImplementedError(
+                    "Get Ryan to make corrections for this type of model in proximal acq, "
```

Comment:
Perhaps put your GitHub name in here @roussel-ryan ?

Reply:
Done